local-cli/templates/HelloWorld/android/build.gradle

```diff
@@ -31,9 +30,3 @@ allprojects {
         }
     }
 }
-
-
-task wrapper(type: Wrapper) {
```

Comment:
Same as above.

Reply:
Overriding wrapper scripts is deprecated and will be removed in future gradle versions